sympy/solvers/inequalities.py

```diff
@@ -1146,4 +1154,9 @@ def linear_programming(A, B, C, D):
         else:
             argmin_dual.append(S.Zero)
 
-    return M[-1, -1], argmax, argmin_dual
+    for i in range(len(argmax)):
+        argmax[i] = nsimplify(argmax[i])
+    for i in range(len(argmin_dual)):
+        argmin_dual[i] = nsimplify(argmin_dual[i])
```

Comment:
I don't think that this should use nsimplify

Reply:
but if I won't use nsimplify the results will be really long and messy (although they are still correct). 
still want me to remove it?

Reply:
Can you give an example of the difference?

Reply:
this one, for example-
Expected:
    (33/4, [33/4, 0, 0], [0, 0, 3/2])
Got:
    (33/4, [8.25000000000000, 0, 0], [0, 0, 1.50000000000000])

(

Reply:
I think that's fine. If floats are used then the output can be floats. I don't know whether the algorithm needs to consider floating point error in the pivoting. If you want exact output you should use exact input.

Reply:
ok. I'll remove it.

Reply:
ok. I'll remove it.